### ISSUE TITLE
[ACTIVE_ARBITRATION_WIN_PREQUAL] SWARM-1700 — exposure-plan first-root authority

### ISSUE THREAD

**Issue by Oleksii-debug** [Oleksii-debug/12-6-ai.#1700]
SWARM_PROTOCOL: SWARM-300-V2
SWARM_CONTROL: #723
SWARM_WORKER_ID: SWARM-1700
STATUS: ACTIVE_NONOWNING_PREQUAL_PENDING_POSTCLAIM_DIRECT_ARBITRATION
LOCAL_FREE_ONLY: true
LIVE_MAIN_AT_CLAIM: caa896eae9ef3030bec7f0fcf16d53bbaf1488ce
CURRENT_INTEGRATION_OWNER: #1677 / lawful successor
SCIENTIFIC_MILESTONE: #548 learned-20M
CI_PRESSURE_AT_CLAIM: GREEN (queued=0, in_progress=0)

PERMANENT_LANE: #7 D06 — Evaluation / Benchmarks / Stage Gates
PERMANENT_WORK_KIND: INDEPENDENT_VERIFY
SELECTED_WORK_KIND: NONOWNING INDEPENDENT VERIFY / PRE-OPTIMIZER AUTHORITY PREQUALIFICATION
SWARM_LANE_KEY: D06|EXPOSURE-PLAN-FIRST-ROOT|INDEPENDENT-VERIFY|PREOPTIMIZER-AUTHORITY-V1
PARENT_ISSUES: #5 #7 #548 #723 #1256 #1336 #1525 #1613 #1698
PRODUCT_MUTATION: false
AUTHORITATIVE_PR_AUDIT_VERDICT: false
MERGE_AUTHORITY: false
TRAINING_CLAIM: false
CLAIMED_AT_UTC: THIS_ISSUE_UPDATED_AT_TIMESTAMP_IS_AUTHORITY
LEASE_UNTIL_UTC: CLAIMED_AT_PLUS_6_HOURS unless terminal/released sooner

ROUTING / FINISH-FIRST BASIS:
- SWARM-1700 maps to permanent lane #7 D06 and work kind INDEPENDENT_VERIFY.
- Binding optimized-target exposure remains exactly zero, so learned-target optimizer step 1 is scientifically prohibited.
- #1677 owns integration; #1621 owns PR1549 changed-head audit; #1630 owns real PR1549 payload execution; #1698 owns the exact D04 token-content same-lineage repair. This claim does not mutate or certify any of those surfaces.
- #1698 terminal diagnostic exposed a distinct secondary authority question after its headline token-content HIGH: `build_deterministic_exposure_plan()` accepts caller-authored full batch/claim grouping/order and self-hashes it; runtime rejects substituted plans only after an expected plan identity has already been selected. #1698 explicitly rotated Product ownership to the disjoint token-content repair.
- Exact preclaim semantic search for this first-root key returned no current owner.

LIVE EXACT TARGETS AT CLAIM:
- current main `caa896eae9ef3030bec7f0fcf16d53bbaf1488ce`;
- canonical current-main `src/twelve_six/data/deterministic_exposure_order.py`: plan builder accepts caller-supplied batch sequence and produces its self-hashed `plan_identity_sha256`; preflight proves ledger/budget/no-overlap/full-one-pass properties but does not by itself establish who selected the first plan;
- readiness PR #1256 currently OPEN/non-draft at exact head `dd905bb39e2685b258c075048b383f12b965886f` and remains Product-owned elsewhere;
- bounded runtime PR #1336 currently OPEN/non-draft at exact head `cfc8c09d09c9ec05a0894e68eacdfcd9f4194f46` and remains Product-owned elsewhere. Its runtime can require an externally supplied exposure-plan root and durable start projection stores that root; this claim therefore asks only about independent provenance of the FIRST root before first-run selection, not the already-existing subsequent-use guard.

BINARY QUESTION AFTER ARBITRATION WIN:
Before optimizer step 1, is there one canonical independently rooted producer/preregistration path that derives the exact first full exposure-plan identity from immutable packing/ledger/recipe/order authority and carries that exact root through readiness/portable run binding into the bounded runtime? Or can a first caller choose between two different structurally valid, complete, replay-free one-pass plans over the same exact ledger, self-hash either plan, then supply the chosen hash as the first expected root and thereby make caller-selected ordering authoritative?

OWNED DIAGNOSTIC SURFACE ONLY:
- read-only exact-source/contract inspection on current main and exact current PR1256/PR1336 heads;
- trace all producers/callers/carriers of `plan_identity_sha256`, `expected_plan_identity_sha256` and `exposure_plan_identity_sha256` from D04 ledger/packing through readiness/trusted bundle/portable packet/D10/run-attempt binding/runtime prehook;
- adversarially establish whether alternate complete valid plan A/B under one ledger can both pass construction/preflight and whether either can become the initial expected root absent an independent preregistered/canonical source;
- distinguish internal self-hash/determinism of worker/shard placement from external authority over the first batch/claim grouping/order;
- publish exactly one source-backed NO_NEW_DEFECT closure or one concrete HIGH/MEDIUM trust-boundary finding plus minimum SAME existing-lineage repair/regression contract to lawful owners;
- no branch, commit, PR, Product mutation, merge, optimizer/model update or final-test access.

NONOVERLAP / AVOID:
- no mutation of #1698 token-content repair files;
- no PR1256 readiness Product mutation (#1599/lawful successor);
- no PR1336 bounded-runtime Product mutation (#1604/lawful successor);
- no PR1525 unique-loss Product mutation;
- no PR1549 audit/execution, central integration, D05 recovery, D06 final-test payload/outcome access;
- no new data/source/readiness/runtime architecture;
- no paid compute, foreign pretrained weights, external LLM/API corpus intelligence.

PACKAGE_DIMENSIONS:
- independent_verification / exact-source review
- adversarial_or_negative proof with alternate valid plan roots
- live exact-head / authority binding
- end-to-end D04 -> readiness -> durable start -> optimizer-boundary trace
- reproducibility / first-run preregistration proof
- operator/Product-owner handoff

ACCEPTANCE:
1. Mandatory postclaim DIRECT OPEN issue collection newest-first at 100/page and separate DIRECT OPEN PR collection newest-first at 100/page both reach exact empty sentinels. Earliest valid same-semantic claimant wins; collision => release before authoritative finding.
2. Final race refresh late-binds current main, PR1256 exact head/owner, PR1336 exact head/owner and #1698 scope before verdict.
3. Enumerate the exact code path that first creates a plan and every externally expected authority root it consumes before that root can reach runtime.
4. Closure requires that the first plan root is derived/preregistered from independently authenticated inputs by a canonical rule/tool and cannot be caller-selected merely by choosing another complete valid order then self-hashing it.
5. If two distinct valid complete plans over the same exact ledger can both pass preflight and the first caller can choose either root without an independent upstream root/canonical order authority, publish a concrete pre-step trust-boundary finding. Minimum repair must extend an existing D04/readiness/run authority lineage rather than create framework #2.
6. Existing runtime enforcement after an expected root exists must be credited; do not misreport that as missing if source proves it.
7. Preserve scientific truth: this diagnostic grants zero corpus/tokenizer/exposure/optimizer/training/learned-weight/final-test credit.
8. Release this semantic immediately after durable evidence and refresh live state. Positive binding optimized-target exposure + terminal runtime/recovery with no training owner preempts lower work with genuine bounded LOCAL_FREE training.

TRUTH_BOUNDARY_AT_CLAIM:
AUTHORIZED_OPTIMIZED_TARGET_EXPOSURE=0
TOKENIZER_FIT_AUTHORIZED=false
OPTIMIZER_UPDATES_EXECUTED_ON_REAL_TARGETS=0
TRAINING_EXECUTED=false
LEARNED_WEIGHTS_CREATED=false
FINAL_TEST_OUTCOMES_READ=false
PAID_COMPUTE_USED=false
FOREIGN_PRETRAINED_WEIGHTS=false
EXTERNAL_LLM_OR_API_USED_FOR_DATA_OR_INTELLIGENCE=false

NEXT NOW:
Exhaust mandatory direct OPEN issue + OPEN PR arbitration to exact empty sentinels and exact-key race refresh. WIN => inspect the exact current source/carrier chain and publish one concrete closure/finding immediately.

**Comment by Oleksii-debug** [comment]
POSTCLAIM DIRECT ARBITRATION — COMPLETE / WIN

SWARM_LANE_KEY: D06|EXPOSURE-PLAN-FIRST-ROOT|INDEPENDENT-VERIFY|PREOPTIMIZER-AUTHORITY-V1

- Direct OPEN issue collection enumerated newest-first at 100/page: pages 1..11 nonempty, page 12 exact `[]`.
- Direct OPEN PR collection enumerated newest-first at 100/page: pages 1..5 nonempty, page 6 exact `[]`.
- Exact-key race refresh after both sweeps returns only SWARM-1700 for this key.
- Newer #1701 is registration-only at the refresh observed during the sweep and does not claim this semantic.
- New PR #1702 belongs to SWARM-1698 and exact key `REPAIR|MAIN|D04-POSTPACK-TOKEN-CONTENT-BINDING|CANONICAL-MATERIALIZATION-V2`; it is the disjoint canonical token-content bridge repair already excluded from this claim.
- No earlier valid same-semantic claimant or open PR owns first exposure-plan root provenance.

ARBITRATION_RESULT: WIN
PRODUCT_MUTATION=false
AUTHORITATIVE_PR_AUDIT_VERDICT=false
MERGE_AUTHORITY=false

Proceeding now with exact-source end-to-end prequalification on current main + exact live PR1256/PR1336 heads. Any target-head movement invalidates the late-bound portion and triggers refresh before verdict.

**Comment by Oleksii-debug** [comment]
TERMINAL PREQUALIFICATION — NO_NEW_DEFECT / OWNERSHIP RELEASED

SWARM_LANE_KEY: D06|EXPOSURE-PLAN-FIRST-ROOT|INDEPENDENT-VERIFY|PREOPTIMIZER-AUTHORITY-V1
ARBITRATION_RESULT: WIN (direct OPEN issues pages 1..11 + page 12 exact `[]`; direct OPEN PRs pages 1..5 + page 6 exact `[]`)
LATE_BOUND_MAIN: caa896eae9ef3030bec7f0fcf16d53bbaf1488ce
TARGET_PR1256_HEAD: dd905bb39e2685b258c075048b383f12b965886f (OPEN/non-draft)
TARGET_PR1336_HEAD: cfc8c09d09c9ec05a0894e68eacdfcd9f4194f46 (OPEN/non-draft)
PRODUCT_MUTATION=false
AUTHORITATIVE_PR_AUDIT_VERDICT=false
MERGE_AUTHORITY=false
TRAINING_EXECUTED=false
FINAL_TEST_OUTCOMES_READ=false

VERDICT: NO_NEW_DEFECT on the claimed FIRST exposure-plan-root trust-boundary semantic.

SOURCE-BACKED TRACE:
1. `build_deterministic_exposure_plan(...)` accepts an explicit batch sequence and self-hashes the resulting complete plan. Preflight then proves ledger coherence, no replay, full one-pass coverage/budget, and can reject any plan whose identity differs from an independently expected `expected_plan_identity_sha256`. This by itself is post-selection enforcement, not the first authority source.
2. Existing D04 adversarial validation intentionally builds another structurally valid complete substitute plan and proves that it is rejected under the original expected plan root. So plan substitution *after* preregistration is already fail-closed.
3. The initial source trace looked suspicious because current-main launch/portable closed-world schemas do not directly expose a top-level exposure-plan root. That suspicion is falsified by the newer exact PR1256 readiness lineage: `preoptimizer_authority.py` roots the exact loss-bearing content manifest under an independently supplied `trusted_readiness_bundle_sha256` and binds that preoptimizer projection into the portable packet. The loss-bearing manifest transitively binds the exact selected exposure-plan identity and ordered loss-bearing claims/content.
4. Exact PR1336 runtime then requires the expected exposure-plan identity, validates preflight and content-manifest coherence against it, carries the identity in the durable start projection, and rechecks the boundary before optimizer work. Therefore the runnable process cannot silently select a second plan after the trusted preoptimizer authority has pinned the manifest/plan root.
5. This matches the earlier independent #1609 carrier conclusion on the PR1256 lineage: D04 exposure state/next-exposure identity plus the loss-bearing manifest already transitively bind exact plan identity/content; PR1256 provides the trusted preoptimizer carrier; the remaining consumer seam belonged to the bounded runtime lineage rather than to a second top-level D04 cursor schema.

DATA-ORDER SEED CHECK:
- canonical LEARN-345 policy does freeze `seed_vector.model_init/data_order/dataloader = 20260826/20260826/20260826` and hashes the complete policy;
- however the binding policy/recipe tests inspected here do not state that `data_order` alone is the canonical producer of the D04 full exposure plan, and the independently trusted D04/D05 next-exposure/manifest/session identities already distinguish different exact plans;
- therefore the fact that `build_deterministic_exposure_plan()` accepts an explicit sequence is not sufficient evidence of a scientific trust-boundary defect: two different preregistered plans do NOT remain the same trusted runnable authority/session. Calling this HIGH would overstate the binding contract.
- If project authority later explicitly declares `seed_vector.data_order` to be the unique normative derivation rule for the D04 plan, the current builder would need a same-lineage canonical derivation regression. No such binding requirement was established by the exact policy/tests/source inspected here, so no launch-blocking defect is claimed now.

SCIENTIFIC CREDIT:
AUTHORIZED_OPTIMIZED_TARGET_EXPOSURE remains 0.
TOKENIZER_FIT_AUTHORIZED remains false.
OPTIMIZER_UPDATES_EXECUTED_ON_REAL_TARGETS remains 0.
TRAINING_EXECUTED=false.
LEARNED_WEIGHTS_CREATED=false.
This closure grants no readiness/training/final-test credit.

OWNERSHIP_RELEASED=true
NEXT: refresh live #723/main/owners/CI and consume the nearest qualified candidate, exact-head audit, real LOCAL_FREE execution gate, or same-lineage repair. Positive optimized-target authority + terminal runtime/recovery still preempts everything with genuine bounded LOCAL_FREE training.

**Comment by Oleksii-debug** [comment]
POST-RELEASE LIVE REFRESH — NO_SAFE_UNCLAIMED_WORK / ZERO-FOOTPRINT CHECKPOINT

SWARM_WORKER_ID: SWARM-1700
PRIOR_CLAIM: D06|EXPOSURE-PLAN-FIRST-ROOT|INDEPENDENT-VERIFY|PREOPTIMIZER-AUTHORITY-V1
PRIOR_RESULT: TERMINAL_NO_NEW_DEFECT_RELEASED
PRODUCT_OWNERSHIP: NONE
AUDIT_OWNERSHIP: NONE
MERGE_AUTHORITY: false
EXECUTION_OWNERSHIP: NONE
TRAINING_CLAIM: false

FRESH LIVE TRUTH:
- exact `main` remains `caa896eae9ef3030bec7f0fcf16d53bbaf1488ce`; latest merge is still PR1372 G06 execution authority/mechanics and explicitly grants no corpus/tokenizer/exposure/optimizer/training/learned-weight/final-test credit;
- PR #1702 exact `ad0b84ca577f9837f190ed69f9c1708c0c5aea6e` remains OPEN, non-draft, mergeable and already has terminal fresh different-worker #1703 `PASS_FOR_INTEGRATION_D04_POSTPACK_TOKEN_CONTENT`; integration belongs to active #1677, so no shadow merge is lawful;
- #1621 still owns PR #1549 exact-head audit `1bf71fd5ae23b138e50805ff3b506094c23b5374`; refreshed comments still show `AUDIT_OWNERSHIP: ACTIVE_READ_ONLY` and no terminal `PASS_FOR_INTEGRATION_POST_G05_G06_MATERIALIZATION / CHANGES_REQUIRED / BLOCKED` verdict;
- #1630 still owns the immediate successor `REAL_POST_G05_G06_PAYLOAD_EXECUTION`; #1625 already proved the 272-record / 5,921,963-byte payload reconstructible and recorded the LOCAL_FREE command, but execution must wait for PR1549 authority/integration;
- PR1336 current Product remains incumbent-owned by #1604 and not released as a stable terminal-green changed head; no fresh audit claim is lawful there;
- newer SWARM-1704 independently terminalized `NO_SAFE_UNCLAIMED_WORK` on the same live main and explicitly yields PR1702 integration, PR1549 audit, PR1549 real execution and incumbent Product lanes. No later collision-safe free critical-path surface is exposed by that refresh.

DECISION:
No second claim is created. The shortest lawful state transitions are already owned:
1. #1677 consume audited PR1702;
2. #1621 terminalize PR1549 exact-head audit;
3. on PASS, #1677 consume/authorize the lineage and #1630 execute the genuine prepared LOCAL_FREE physical payload;
4. then consume existing downstream gates in binding order;
5. only after positive binding optimized-target exposure plus terminal tokenizer/unique-loss/readiness/runtime/recovery/run authority, claim and execute genuine bounded LOCAL_FREE learned-target training if no training owner exists.

SCIENTIFIC BOUNDARY:
AUTHORIZED_OPTIMIZED_TARGET_EXPOSURE=0
TOKENIZER_FIT_AUTHORIZED=false
OPTIMIZER_UPDATES_EXECUTED_ON_REAL_TARGETS=0
TRAINING_EXECUTED=false
LEARNED_WEIGHTS_CREATED=false
FINAL_TEST_OUTCOMES_READ=false
PAID_COMPUTE_USED=false
FOREIGN_PRETRAINED_WEIGHTS=false
EXTERNAL_LLM_OR_API_USED_FOR_DATA_OR_INTELLIGENCE=false

WAKE-UP TRIGGERS:
- `main` moves after #1677 consumes PR1702 or another qualified terminal candidate -> refresh exact downstream identities and take only a newly unowned transition;
- #1621 publishes terminal PR1549 PASS -> do not duplicate #1630; verify #1677/#1630 consume it, and take the nearest newly released disjoint gate if one exists;
- any incumbent Product releases a stable GREEN changed head needing a fresh different-worker audit -> claim only after exact collision arbitration;
- positive binding optimized-target exposure + all exact terminal training prerequisites + no exact training owner -> genuine LOCAL_FREE training immediately preempts diagnostic work.

FINAL_THIS_PULSE: NO_SAFE_UNCLAIMED_WORK. This is not project completion and not a trained-model claim; it is a collision-safe release after consuming the available nonowning diagnostic and refreshing the next real-state transitions.
